pennylane/templates/broadcast.py

```diff
@@ -566,5 +566,7 @@ def circuit(pars):
     }
 
     # broadcast the unitary
-    for wires, pars in zip(wire_sequence[pattern], parameters):
-        unitary(*pars, wires=wires, **kwargs)
+    wire_sequence = pattern_to_wires[pattern]
+    for i in range(len(wire_sequence)):
+        # note: iteration over indices is faster in tensorflow than iteration over parameters
+        unitary(*parameters[i], wires=wire_sequence[i], **kwargs)
```

Comment:
Will the star unpack the tensor anyways? Is this maybe why no speedup is measurable from this change?

Reply:
Ah, this might be the case! @trbromley can probably fill us in with more details, as he first reported this issue